ckanext/ontario_theme/fanstatic/external/ontario_theme.css

```diff
@@ -775,7 +775,14 @@ div.add-dataset {
   position: relative;
   border-bottom: 1px solid #dddddd;
 }
-.media-item:last-child {
+.ministry-item {
+  margin-left: 5px;
+  padding-left: unset;
+}
+.ministry-item.media-item:first-child{
+  border-top: 1px solid #dddddd;
+}
+.media-item:last-child{
   border-bottom: none;
```

Comment:
On data.ontario.ca site there is a double border because of the bottom border of the last child. 
![Screenshot from 2023-01-03 11-59-21](https://user-images.githubusercontent.com/60950452/210404469-4ccad32d-ef56-403a-9fca-a1d3061da432.png)
This was fixed on the test site 
![Screenshot from 2023-01-03 12-00-57](https://user-images.githubusercontent.com/60950452/210404693-48412986-ff64-4995-9915-6f839e71a539.png)
The bottom border of the last child shouldn't be showing, but the top border of the pagination should be.

Reply:
When there is no pagination (for a small number of files), the bottom border appears after the last child. Should it be there?
<img width="1198" alt="Screen Shot 2023-01-04 at 9 15 13 AM" src="https://user-images.githubusercontent.com/1254764/210574191-ac0ec6e6-14cc-447a-8dc8-65e11358658f.png">

Reply:
I tried on both Firefox and Chrome and I wasn't seeing the bottom border on my end